main.go

```diff
@@ -14,13 +17,36 @@ var (
 
 func main() {
 	app := cli.NewApp()
-	app.Name = "rancher-vxlan"
 	app.Version = VERSION
 	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "listen",
+			Value: ":8111",
+		},
 		cli.BoolFlag{
 			Name:   "debug, d",
 			EnvVar: "RANCHER_DEBUG",
 		},
+		cli.StringFlag{
+			Name:   "bridge",
+			Value:  vxlan.DefaultBridgeName,
+			EnvVar: "RANCHER_BRIDGE",
+		},
+		cli.StringFlag{
+			Name:   "metadata-ip",
```

Comment:
Initially I started out with "metadata-ip", but later switched to "metadata-address" and env: "RANCHER_METADATA_ADDRESS".